src/ImageSharp/Formats/Jpeg/Components/Block8x8F.cs

```diff
@@ -421,16 +488,17 @@ public void LoadFromInt16ExtendedAvx2(ref Block8x8 source)
     /// <param name="value">Value to compare to.</param>
     public bool EqualsToScalar(int value)
     {
+        // TODO: Can we provide a Vector128 implementation for this?
```

Comment:
What's blocking a V128 path from being added? At a glance it looks like it should be almost a copy/paste of the V256 path...